pkg/descheduler/strategies/nodeutilization/highnodeutilization.go

```diff
@@ -29,8 +29,48 @@ import (
 	"sigs.k8s.io/descheduler/pkg/descheduler/evictions"
 	nodeutil "sigs.k8s.io/descheduler/pkg/descheduler/node"
 	podutil "sigs.k8s.io/descheduler/pkg/descheduler/pod"
+	"sigs.k8s.io/descheduler/pkg/utils"
 )
 
+// balanceNodes balance nodes from low utilized nodes to high utilized nodes. A corner case: All nodes are underutilized
+func balanceNodes(sourceNodes, destinationNodes []NodeInfo) ([]NodeInfo, []NodeInfo) {
+	if len(sourceNodes) == 0 || len(destinationNodes) > 0 {
+		return sourceNodes, destinationNodes
+	}
+	numOfBalanced := 0
+	partition := int(float64(len(sourceNodes)) * 0.5)
+	src := make([]NodeInfo, 0, partition)
+	dst := destinationNodes
+	hasImportantPods := func(node *NodeInfo) bool {
```

Comment:
`hasImportantPods` might be a generic function. @damemi @a7i @knelasevero worth discussing definition of what an important pod is.

Reply:
What we have here seems like a good start I think. Mirror, with PVC, or Critical.